tests/local_users_passwords_reset_test.py

```diff
@@ -0,0 +1,34 @@
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+from utilities_common.db import Db
+
+
+class TestLocalUsersPasswordsReset:
```

Comment:
could you add config db json for test as well? https://github.com/sonic-net/sonic-utilities/tree/master/tests